1001-2000/LIT1473GadlaL.xml

```diff
@@ -84,7 +87,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </bibl>
             </listBibl>
          </div>
-         <div type="edition">
+         <div type="edition" xml:id="recension2">
+            <note>Recension 2 provided after <bibl>
+               <ptr target="bm:Bausi2003Libanos"/>
+            </bibl></note>
             <div type="textpart" subtype="incipit" xml:lang="gez">
                <ab>
                 ስምዑ፡ እንግርክሙ፡ ዘንተ፡ ነገረ፡ በእንተ፡ <gap reason="omitted"/> ዘከመ፡ ሕይወ፡ ውስተ፡ ምድር፡ ፭፻፷ወ፯፡ ዓመተ፡ <gap reason="omitted"/> 
```

Comment:
Please check the word ሕይወ፡

Reply:
It should be ሐይወ፡

Reply:
This incipit was not inserted by me as you can see from the file, this is an old record by MV, I don't have Bausi's edition at hand now - can you check please?

Reply:
Should I type it in?